src/http-util/src/lib.rs

```diff
@@ -19,3 +19,65 @@ where
 {
     Html(template.render().expect("template rendering cannot fail"))
 }
+
+#[macro_export]
+/// Generates a `handle_static` function that serves static content for HTTP servers.
+/// Expects three arguments: an `include_dir::Dir` object where the static content is served,
+/// and two strings representing the (crate-local) paths to the production and development
+/// static files.
+macro_rules! make_handle_static {
```

Comment:
This no longer needs to be a macro, right? It can just be a normal function?

Reply:
We spoke offline. For posterity -- it's difficult to make this a macro due to async function types being unnameable, and we will not let that block this diff.

Nikhil has said he might still try to make it work in the future, for funsies.